index.src.html

```diff
@@ -17,96 +17,99 @@
-    through browser script. Conceptually, one or more credentials are stored on an authenticator, and each credential is scoped
-    to a single Relying Party. Authenticators are responsible for ensuring that no operation is performed without the user's
-    consent. The user agent mediates access to credentials in order to preserve user privacy. Authenticators use attestation
-    to provide cryptographic proof of their properties to the relying party. This specification also describes a functional
-    model of a WebAuthn compliant authenticator, including its signature and attestation functionality.
+Abstract: This specification defines a Web API enabling the creation and use of  strong, attested, cryptographic, scoped credentials
+    by web applications, for the purpose of strongly authenticating users. Conceptually, one or more credentials, scoped
+    to a given Relying Party, are created and stored on an authenticator. Authenticators are responsible for ensuring that no operation is performed without user 
+    consent. The user agent mediates access to credentials in order to preserve user privacy. Authenticators provide cryptographic proof of their properties to relying parties via attestation. This specification also describes the functional
+    model for WebAuthn conformant authenticators, including their signature and attestation functionality.
 Boilerplate: omit conformance, omit feedback-header
 Markup Shorthands: css off, markdown on
 </pre>
 
 
-# Use Cases # {#use-cases}
+# Introduction # {#intro}
 
 [INFORMATIVE]
 
-This specification defines an API for web pages to access scoped credentials through JavaScript, for the purpose of strongly
-authenticating a user. Scoped credentials are always scoped to a single <a>[RP]</a>. This scoping is enforced jointly by the
-User Agent implementing the Web Authentication API and the authenticator that holds the credential, by constraining the
-availability and usage of credentials. Scoped credentials created by a [RP] can only be accessed by web origins belonging to that
-[RP]. Additionally, privacy across [RPS] must be maintained; scripts must not be able to detect any properties, or even the
-existence, of scoped credentials belonging to other [RPS].
+This specification defines a Web API enabling the creation and use of strong, attested, cryptographic, <em><a>scoped credentials</a></em> by web applications, 
+for the purpose of strongly authenticating users. 
+A <a>scoped credential</a>, created and stored, subject to <em><a>user verification</a></em>, by an <em><a>authenticator</a></em> at the behest of a <em><a>[RP]</a></em>, can only be accessed by web origins belonging to that [RP]. 
+This scoping is enforced jointly by <em><a>conforming User Agents</a></em> and <em><a>authenticators</a></em>. 
+Additionally, privacy across <a>[RPS]</a> is maintained; [RPS]' browser scripts are not able to detect any properties, or even the existence, of credentials scoped to other [RPS].
```

Comment:
whoops, actually, 'browser script' is not held over, so this should be addressed.